cetmix_tower_server/tests/test_command.py

```diff
@@ -38,11 +38,16 @@ def setUp(self, *args, **kwargs):
             }
         )
 
+        # secret value as multi line string
         self.python_ssh_key = self.Key.create(
             {
                 "name": "Test Python SSH Key",
                 "key_type": "k",
-                "secret_value": "Python much key",
+                "secret_value": """
```

Comment:
Let's add the key reference explicitly to ensure that is always remains the same. Like [here](https://github.com/cetmix/cetmix-tower/blob/f3b91c1f352e738a3bf120f7bc549d177c36cb84/cetmix_tower_server/tests/test_command.py#L130C16-L130C50). For not to depend on the possible changes is the reference generation engine.

Reply:
I don't quite understand you, this example is the key that contains the multiline string, it is used in the command in which you gave the link as an example.

Reply:
Just add the reference explicitly:
```python
...
"reference": "test_python_ssh_key",
..
```